clipper_admin/clipper_manager.py

```diff
@@ -371,6 +368,52 @@ def register_application(self, name, model, input_type, default_output,
         print(r.text)
         return r.status_code == requests.codes.ok
 
+    def link_model_to_app(self, app_name, model_name):
+        """Allows the model to be used by app
+
+        Parameters
+        ----------
+        app_name : str
+            The name of the application
+        model_name : str
+            The name of the model to link to the application
+
+        Returns
+        -------
+        bool
+            Returns true iff the model link request was successful
+        """
+        url = "http://%s:%d/admin/add_model_links" % (self.host,
+                                                      CLIPPER_MANAGEMENT_PORT)
+        req_json = json.dumps({
+            "app_name": app_name,
+            "model_names": [model_name]
+        })
+        headers = {'Content-type': 'application/json'}
+        r = requests.post(url, headers=headers, data=req_json)
+        print(r.text)
+        return r.status_code == requests.codes.ok
+
+    def get_linked_models(self, app_name):
```

Comment:
This results in `ValueError: No JSON object could be decoded` in the case that `app_name` is not a registered application.